docs/service-release-checklist.md

```diff
@@ -0,0 +1,104 @@
+# Service readiness checklist
+
+This document aims to provide a checklist for determining whether service is mature enough for general availability.
+
+## Checklist
+
+Legend: `M` - Mandatory, `R` - Recommended, `O` - Optional
+
+### Documentation
+
+- `R`: README.md in the repository:
+  * service description, ownership, links to documentation,
+    build and deployment instructions, configuration variables, healthchecks, etc
+- `R`: Architectural diagram for the service
+- `O`: SLI/SLO/SLA definitions (at least service criticality level: P1/2/3)
```

Comment:
Marking this as recommended would also bring some awareness into the reliability of the service. Starting from the basics, like setting an acceptable error rate and latency is not that hard.